src/easydmp/plan/models.py

```diff
@@ -21,6 +24,8 @@
 from .utils import purge_answer
 from .utils import get_editors_for_plan
 
+if TYPE_CHECKING:
+    from easydmp.auth.models import User
```

Comment:
What's the point of this?

Reply:
This prevents import of User in running code, the import is only done when checking annotations. I should probably have used `get_user_model()` instead of an import though. Note that the user-model is so-called "replaceable" and the rest of the code uses several techniques to avoid importing it directly for that reason.

Reply:
That is the reason I put user: Any to begin with. The cruft above is hardly worth it.